conda_index/index/convert_cache.py

```diff
@@ -148,15 +148,15 @@ def remove_prefix(conn: sqlite3.Connection):
     log.info("Migrate database")
 
     def basename(path):
-        if not isinstance(path, str):
+        if not isinstance(path, str):  # pragma: no cover
             return path
         return path.rsplit("/")[-1]
 
     try:
         conn.create_function(
             "migrate_basename", narg=1, func=basename, deterministic=True
         )
-    except TypeError:  # Python < 3.8
+    except TypeError:  # pragma: no cover; Python < 3.8
```

Comment:
We no longer support Python <3.8 so we can drop this?

Reply:
Let's

Reply:
Done